### ISSUE TITLE
Adjusting PR boilerplate and labels to improve Release Notes UX

### ISSUE THREAD

**Issue by onyiny-ang** [kubernetes/sig-release#668]
There was some discussion during this release cycle and in [slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1560535286311000 ) about collecting more information from each PR in order to improve the user experience for release notes and ease curation for the release notes team.
This boils down to 3 main improvements:
1. Collecting `KEP` numbers associated with any PRs that work towards implementing a KEP.
2. Collecting the `Primary-sig` or sponsoring SIG associated with each PR to eliminate guess work for the release notes team.
3. Labeling the PR as a deprecation if it is a deprecation to a dependency.

I would like to propose a subform--or in any case, more detailed release note section of the k/k PR boilerplate that ensures a contributor fills out the `Primary SIG` section and has a space to associate a `KEP` # if it is appropriate. A label to indicate deprecations would be great.

**Comment by onyiny-ang** [comment]
/cc @tpepper @justaugustus @jeefy @saschagrunert @lachie83 @jberkus 
/sig release

Ideally we would like to have some (or all) of these things available starting in the 1.16 release

**Comment by saschagrunert** [comment]
I think the only related PR is https://github.com/kubernetes/kubernetes/pull/81278, which we either have to finish or postpone to the next release.

**Comment by saschagrunert** [comment]
Please take a look at the two linked PRs, we decided to split up the work a bit more.

**Comment by guineveresaenger** [comment]
We are waiting on some automation and decisions around proposed labels here.

This won't make the 1.17 milestone.

/milestone v1.18

**Comment by justaugustus** [comment]
I think we can close the loop on this before 1.17 is out.
/milestone v1.17

**Comment by saschagrunert** [comment]
I propose that we close this issue now. If we're going to sort the notes by kind then we would not need a primary SIG at all, WDYT?

**Comment by justaugustus** [comment]
This needs to stay open until we have a community decision on kind vs SIG.
See here: https://github.com/kubernetes/release/pull/923#issuecomment-558880526

I'll send a note out next week since US folks are on holiday now.
/remove-help
/unassign
/area release-eng

**Comment by saschagrunert** [comment]
@justaugustus I think we can close this issue, what do you think?

**Comment by alejandrox1** [comment]
Just confirmed with @justaugustus that it is cool to close this one
/close
